Lib9c/Model/State/WeeklyArenaState.cs

```diff
@@ -112,7 +112,34 @@ private void ResetOrderedArenaInfos()
         }
 
         /// <summary>
-        /// 인자로 넘겨 받은 `avatarAddress`를 기준으로 상위와 하위 범위에 해당하는 랭킹 정보를 얻습니다.
+        /// Get arena rank information.
+        /// </summary>
+        /// <param name="firstRank"></param>
+        /// <param name="count"></param>
+        /// <returns></returns>
+        public List<(int rank, ArenaInfo arenaInfo)> GetArenaInfos(
+            int firstRank = 1,
+            int? count = null)
+        {
+            if (firstRank < 1 ||
+                firstRank > OrderedArenaInfos.Count)
```

Comment:
it's a quite more readable.
```suggestion
            if (!(0 < firstRank && firstRank <= OrderedArenaInfos.Count))
```

Reply:
Oh, too, when I checked the range, I preferred to write it in that structure that is visually more readable.
However, I have previously accepted the opinion that it is better to have the object to be examined on the left side.
The latter feels like a convention, but I think the visually readable structure is better.
In the future, I will write it in a structure that is visually easy to read.